### PR TITLE
Adding new files to comply with github community guidelines

### LINKED ISSUE #2045: complete community profile
to conform to community standards, this repository needs: 

- [x] a code of conduct

- [x] contributing guidelines

- [x] issue templates

- [x] PR templates

the goal is to echo the `https://github.com/astropy/astropy` guidelines

### PR DESCRIPTION
**Pull request by cmagness**
Adding code of conduct, contribution guidelines, issue & PR templates, and also adding link to new files from README.

Should close #2045.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+Contributing to `glue` (Adapted from [`astropy`](https://github.com/astropy/astropy))
+================================================
```
Comment (astrofrog) [inline CONTRIBUTING.md]: I think the Adapted from astropy bit could go in a footer at the bottom, it doesn't need to be in the title

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)
+- [ ] I have read the [Contribution Guidelines](CONTRIBUTING.md)
+- [ ] My code follows the [`astropy` style guidelines](http://docs.astropy.org/en/latest/development/codeguide.html) used for this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (astrofrog) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think this template is a bit too detailed, I think I would be happy with making sure they have a good description and the 'type of change' section but I think the rest might be a bit too much (I get a bit scared/put off by detailed templates like this for other packages)

**Comment by cmagness** [comment]
@astrofrog i've made the changes as you suggested -- to reformat the PR template i removed the test section and reduced the checklist significantly

**Comment by astrofrog** [comment]
@cmagness - thanks and sorry for the delay! I just realized that this would be useful for all the other repos in the organization, so I think it might make more sense to add many of these files to this repository:

https://github.com/glue-viz/.github

which should then impact all repositories? (I found out about this approach on https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization#creating-a-repository-for-default-files).

**Pull request merged**